app/Http/Controllers/Service/DashboardController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Controllers\Service;
+
+use App\Http\Controllers\Controller;
+use Auth;
+
+class DashboardController extends Controller
+{
+    /**
+     * Index the Dashboard options
+     */
+    public function index()
+    {
+        $user = Auth::user();
+        $data = [
+            "user_name" => $user->name,
+            "centre_name" => $user->centre->name,
```

Comment:
centre_id is nullable. Do we need to make sure this user has one?

Reply:
We do, I'll change it.